res/ai/DefaultAIPlayer/CommonObjects.lua

```diff
@@ -480,6 +485,17 @@ function AIToolsClass:GetBroadcastAttraction(broadcastMaterialSource, day, hour,
 			result = result * relTop
 		end
 	end
+	local timesShown = broadcastMaterialSource:GetTimesBroadcasted(forPlayer)
```

Comment:
Vielleicht sollten wir solche Codebloecke mit "fixen Werten" irgendwie markieren ... so dass wir spaeter "Balancing"-Codestellen schneller wiederfinden.
Hier koennte auch eine "KI-Charakterzug" die Werte leicht anpassen (fuer Testlaeufe kann man dann gezielt Varianten erschaffen - bzw alle erzwungen "gleich" handeln lassen).

Reply:
Ja. Im Prinzip müsste man diesbezüglich ein komplettes Review durchführen. An einigen Stellen habe ich ja TODOs dringelassen, damit man die Stellen wiederfindet. Aber fixe Werte gab es auch vorher schon.
Ich habe ein solches Review auf dem Schirm.
Was man kurzfristig (nicht in diesem PR) machen könnte, wäre alle solche Zahlen als Konstanten in die Strategie zu stecken. Dann hätte man eine zentrale Anlaufstelle für solche Werte, könnte nach der Verwendung suchen und optimieren.
Oftmals sind es nicht einemal Charakterzüge, die für den Wert verwantwortlich sein sollten, sondern "Gelerntes" (mit diesem Genre habe ich bislang nur geringe Einschaltquoten gehabt - also häufiger verkaufen, mit geringerer Wahrscheinlichkeit wieder kaufen).